cognite/extractorutils/unstable/core/runtime.py

```diff
@@ -419,6 +424,45 @@ def run(self) -> None:
 
         self.logger.info(f"Started runtime with PID {os.getpid()}")
 
+        if args.service and sys.platform == "win32":
```

Comment:
This is a tiny thing, but maybe we could check the `service` flag on non-windows and throw an exception?